assets/base.css

```diff
@@ -96,7 +96,7 @@ body::-webkit-scrollbar {
 }
 
 html {
-  overflow-x: hidden;
+  overflow-x: clip;
```

Comment:
Do we really want to clip content that's overflowing the document?

Reply:
Otherwise, we get a horizontal scroll like this

![Arc_2024-02-29 16-12-43@2x](https://github.com/booqable/impact-theme/assets/40244261/d192c695-d2f1-4612-a1b3-d463add058b9)